Mixmatch/open_image_dataset/README.md

```diff
@@ -0,0 +1,43 @@
+# Experiment on Open Image Dataset v6 (OIDv6)
+## Setup
+
+1. Install [OIDv6 tool](https://github.com/DmitryRyumin/OIDv6) to download specific categories from the whole dataset.
+	``` pip install oidv6```
+2. Navigate to /Mixmatch/open_image_dataset directory
+3. Download some categories from the dataset to current directory. (This may take a while)
+	Take Motorcycle, Parking Meter, Stop Sign, Fire Hydrant these 4 categories as an example:
+	```oidv6 downloader --dataset . --type_data all --classes "Motorcycle" "Parking meter" "Stop sign" "Fire hydrant" --yes```
+4. Run img2tfrecord.py to generate tfRecord files from the downloaded images.
+	```python3 img2tfrecord.py```
```

Comment:
nit: add a period at the end to separate from the next sentence.

Reply:
Done